solidity/interfaces/IOngoingAirdrops.sol

```diff
@@ -22,6 +22,16 @@ interface IOngoingAirdrops {
     uint256 amount;
   }
 
+  /**
```

Comment:
As a side note, should we rename all fils from `OngoingAidrops` to `OngoingCampaings`?